_data/tools.yml

```diff
@@ -134,6 +134,25 @@
       url: https://docs.getdbt.com/reference/resource-configs/trino-configs
     - urltext: dbt-trino plugin
       url: https://github.com/starburstdata/dbt-trino
+- name: dolphinscheduler
+  anchor: dolphinscheduler
+  category: client
+  logo: /assets/images/logos/apache-dolphinscheduler.png
+  logosmall: /assets/images/logos/apache-dolphinscheduler-small.png
+  description: |
+    Apache DolphinScheduler is an open-source, distributed workflow scheduling
+    platform designed to manage and execute batch jobs, data pipelines, and ETL
+    processes. DolphinScheduler enables users to create and manage consecutive
+    jobs run easily, including support for different types of tasks, such as SQL
+    statements, shell scripts, Spark jobs, Kubernetes deployments, and many
+    others. 
+  links:
+    - urltext: Apache DolphinScheduler
+      url: https://dolphinscheduler.apache.org/
+    - urltext: Apache DolphinScheduler documentation
```

Comment:
Lets leave the documentation link out for now. We should ping them to add a Trino specific page..